src/main/java/graphql/analysis/QueryVisitor.java

```diff
@@ -16,4 +16,8 @@
 
     void visitFragmentSpread(QueryVisitorFragmentSpreadEnvironment queryVisitorFragmentSpreadEnvironment);
 
+    default void visitFragment(QueryVisitorFragmentEnvironment queryVisitorFragmentEnvironment) {
```

Comment:
I think we should call it `visitFragmentDefinition`. `visitFragment` is not clear enough.